src/pages/token-auth-rules/index.md

```diff
@@ -3,10 +3,42 @@ title: Overview
 metaTitle: Token Auth Rules - Overview
 description: Provides a high-level overview of NFT permissions.
 ---
+Token Authorization Rules (aka Token Auth Rules) is an advanced metaprogramming tool meant to evaluate permissions of an instruction occurring on an SPL Token. The program itself can be used to create and update Rule Sets, which are collections of Rules which represent specific criteria.
 
-_Coming soon..._
+## Introduction
+When a token operation is performed, the program can be called with instruction details (e.g. destination address for a token transfer), and those details are validated against the Rule Set. If the Rules are evaluated to be invalid, the instruction will fail and the whole transaction will be reverted. This enables whole transactions to be built that couple token operations with the Token Auth Rules program so any transactions, and therefore the contained token operation, will be reverted if the Rules in the associated Rule Set are violated.
 
-In the meantime, you may be interested in the following resources:
+## Features
+
+[Create or Update Rule Sets](/token-auth-rules/create-or-update) - An instruction used to both initialize and update Rule Set contents.
+
+[Rule Set Buffers](/token-auth-rules/buffers) - How large Rule Sets are handled.
+
+[Validate Rule Sets](/token-auth-rules/validate) - How a Rule Set is validated.
+
+## Rule Types
+Authorization rules are variants of a `Rule` enum that implements a `validate()` method.
+
+There are **Primitive Rules** and **Composed Rules** that are created by combining of one or more primitive rules.
```

Comment:
I still think there should be consistent terminology throughout, picking one: "Composed" or "Composite".  It's called composed here but composite elsewhere.